code/datums/uplink_items/uplink_general.dm

```diff
@@ -1,4 +1,5 @@
 GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
+#define UPLINK_SPECIAL_SPAWNING "ONE PINK CHAINSAW PLEASE"
```

Comment:
Can you document this define and how it's used?

Reply:
Never done much like this, is this acceptable?